lang/cs/Org.Apache.REEF.Driver/Bridge/ClrHandlerHelper.cs

```diff
@@ -55,27 +77,43 @@ public static ulong CreateHandler(object handler)
             return ul;
         }
 
+        /// <summary>
+        /// Frees a .NET handle.
+        /// </summary>
+        /// <param name="handle">The handle to free</param>
         public static void FreeHandle(ulong handle)
         {
             GCHandle gc = GCHandle.FromIntPtr((IntPtr)handle);
             gc.Free();
         }
 
+        /// <summary>
+        /// Sets the memory granularity in megabytes for the Evaluator Descriptors.
+        /// </summary>
+        /// <param name="granularity">The memory granularity in megabytes</param>
         public static void SetMemoryGranuality(int granularity)
```

Comment:
I'll make it obsolete for now. It seems like ``MemoryGranularity`` is only used to compare memories for ``EvaluatorDescriptorImpl``. In reality I don't think we should be mucking with the numbers, and we should use whatever memory granularity is imposed by the underlying Resource Manager.